entity-types/ext-triton/summary_metrics.yml

```diff
@@ -0,0 +1,34 @@
+nv_cpu_memory_total_bytes:
```

Comment:
Use also camelcase

Reply:
Hi @lujop,
As per your suggestions, I have made changes. Could you please review it.